examples/with-authjs/app.config.ts

```diff
@@ -1,3 +1,5 @@
 import { defineConfig } from "@solidjs/start/config";
 
```

Comment:
@OrJDev , the @solid-mediakit/auth-plugin is added in the list of dependencies, but it's not included in the app.config.ts here like shown in this [docs of protected$](https://mediakit-taupe.vercel.app/auth/protected). Where is it used? Is it necessary?